### PR TITLE
Update controllers.py

### PR DESCRIPTION
**Pull request by kheeong**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [x] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

{pull request content here}

### CODE REVIEW

**Comment by thealphadollar** [comment]
@kheeong Thank you for sending a Pull Request.

Can you please state the purpose of this Pull Request?

**Comment by kheeong** [comment]
> @kheeong Thank you for sending a Pull Request.
> 
> Can you please state the purpose of this Pull Request?

This is a Task from @canihavesomecoffee for the google code in 2019

**Comment by kheeong** [comment]
> @kheeong Thank you for sending a Pull Request.
> 
> Can you please state the purpose of this Pull Request?

This is request is to solve a bug where the session cookies was not deleted/clear even the user log out of the website. This cause the user can log in to the website by just copy their session cookies and paste in without even typing their email and password.

File: mod_auth/controllers.py [hunk 1]
```diff
@@ -452,7 +453,9 @@ def logout():
 
     Return user to the login page.
     """
+
```
Comment (canihavesomecoffee) [inline mod_auth/controllers.py]: Can you remove this line? It's not allowed by pydocstyle.

**Pull request merged**